bad_connector_woocommerce/tests/test_woo_backend.py

```diff
@@ -28,6 +33,7 @@ def setUp(self):
                 "tracking_info": True,
                 "warehouse_id": warehouse.id,
                 "update_stock_inventory": True,
+                "access_token": "d4ea64d3-8f85-4955-be49-4aeb29151801",
```

Comment:
Don't we have to add test_access_token as well, to cover the tests

Reply:
you have skipped this comment